### DISCUSSION TITLE
curl https://ipleak.net/json/

### DISCUSSION THREAD

**Discussion by valdemirjds** [nicolargo/glances#2731] [General]
Hi,
Is there any way to put the "country_name" field in the "Sensors" area ?

curl https://ipleak.net/json/

{
    "as_number": 212238,
    "isp_name": "Datacamp Limited",
    "country_code": "CH",
    "country_name": "Switzerland",
    "region_code": "ZH",
    "region_name": "Zurich",
    "continent_code": "EU",
    "continent_name": "Europe",

Thanks

**Comment by nicolargo** [comment]
Hi @contracer 

the contry of your public IP address can already be displayed in the header thanks to the Censys online service (see see issue #2105).

I did not know ipleak but it looks great. I am going to refactor the IP plugin in order to make the public information service configurable.

Nicolas
